src/Ladybug_Capture View.py

```diff
@@ -130,14 +132,24 @@ def viewCapture(fileName, directory, viewNames, image_width, image_height, dispM
                 pass
         
         return fullPath
+        
 
 if _capture and _fileName!=None:
     
     directory = mdPath(folder_)
     
+    doc = GH.Instances.ActiveCanvas.Document.Objects;
+    isAtLast = doc[doc.Count-1].InstanceGuid.Equals(ghenv.Component.InstanceGuid);
+    if (not isAtLast):
+        # bring this to the top
```

Comment:
Hi @MingboPeng! Thanks. Looks great! There are two/three styling issues which should be very easy to fix.
1. No ; at the end of line 141 and 142.
2. No ( ) after if in line 143.
I blame both on C#  and javascript ;)